webapp/components/comments/CommentList/index.vue

```diff
@@ -47,7 +47,8 @@ export default {
   },
   watch: {
     Post(post) {
-      this.comments = post[0].comments || []
+      const [first] = post
+      this.comments = (first && first.comments) || []
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-11T14:48:23Z" title="Thursday, July 11th 2019, 4:48:23 pm +02:00">Jul 11, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -47,7 +47,8 @@ export default {
   },
   watch: {
     Post(post) {
-      this.comments = post[0].comments || []
+      const [first] = post
+      this.comments = (first && first.comments) || []
```


I saw that you pinged me on the commit that introduced this... 
this protects against calling `.comments` on a null `post[0]` correct?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-11T19:58:24Z" title="Thursday, July 11th 2019, 9:58:24 pm +02:00">Jul 11, 2019</time>_

---


I just saw some JS errors in the console of cypress tests. It did not crash the cypress test though. I couldn't find a solution how we can tell cypress to always crash if there are javascript runtime errors.

So all I did was to fix this without any further ado.